src/ui/projectmanager/pms/pm.go

```diff
@@ -304,16 +304,12 @@ func (p *ProjectManager) GetPublic() ([]*models.Project, error) {
 // GetByMember ...
 func (p *ProjectManager) GetByMember(username string) ([]*models.Project, error) {
```

Comment:
Probably we should move "GetMyProjects" to security context so in admiral case there's no dependency on PM.